scripts/superdesk-workspace/content/content.js

```diff
@@ -84,6 +84,51 @@
                 return _item;
             });
         };
+
+        /**
+         * Create new item using given content type
+         *
+         * @param {Object} contentType
+         * @return {Promise}
+         */
+        this.createItemFromContentType = function(contentType) {
+            var item = {
+                type: TEXT_TYPE,
```

Comment:
We are treating `text only` and `rich text (with images, embeds)` items as same. I like the idea.
But, what would be the profile of ingested items? Can we change the profile after ingestion? How do we treat the associated item in NewsML/NITF?